### DISCUSSION TITLE
channel open still pending after 28 days:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#7737] [Troubleshooting] [answered]
We have a new Channel open that is still pending after 28  days..

Mempool transcation:

https://mempool.space/tx/23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6

We have tried one fee bump using RTL and it appeared to us that it has been accepted but no mention of any change on the mempool as of today?

We realize that at the time of the channel's initial opening we should have checked the current acceptable channel open fee rates.

214 sats was way too low at this point in time.

---

Here is a reply from this channels proposed peer;

The transaction [23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6](https://mempool.space/tx/23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6) is stuck in the mempool due to a low fee rate (see below: 1.01 sat/vB). Check the fee bumping options on your side. From the NiceHash LN node perspective the channel is closed => type: FUNDING_CANCELED.

--

Please let us know what we can do to either have the channel opened or have the funds returned to our wallet so we can better allocate the funds towards a new channel using the appropriate fee rate.

Thank you all.

Orange Bonzo

**Comment by sputn1ck** [comment]
Hey,

I'm not sure about all the possibilities you have, some ideas:

- Force closing the channel. AFAIK this should straight up just work and will give you your funds back, but might come with a large fee
- RBFing the inputs.

In order for the 2nd step to work, we would want to try to create a transaction which spends both inputs to a new change address. If we're lucky this might not be to hard.

As a first step could you try to run `lncli listunspent` and see if the inputs to your channel opening are shown.

**Reply by sputn1ck** [reply]
I opened this PR https://github.com/lightninglabs/chantools/pull/70 which would allow you to rbf your inputs to a new address. If you need some advice on how to proceed you can contact me on kon@lightning.engineering

**Reply by sputn1ck** [reply, marked as answer]
We were able to rbf the inputs by using the `doublespendinput` command from https://github.com/lightninglabs/chantools

**Comment by BonzoDogDooDah** [comment]
The abandon channels command did work?

--

***@***.***:~$ `lncli abandonchannel
> --chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1`
bash: --chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1: command not found
bash: NAME:: command not found

-

Any ideas please, thank you

Orange Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



9 Jun 2023, 17:40 by ***@***.***:

**Reply by guggero** [reply]
You need to put the whole command on a single line, otherwise the shell is going to interpret each line as a new command.

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

You can now run the following command:
`lncli abandonchannel --chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1`

Sorry I am a bit of a newbie.

Can you show me the right way to run this command please?

I am using Lightning Shell in Umbrell and it said this wwhen doing it in one line.

--

***@***.***:~$ lncli abandonchannel --output_index value 23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:0 
Incorrect Usage: invalid value "value" for flag -output_index: parse error

--

Thank you so much for your support.

Orange Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



9 Jun 2023, 18:17 by ***@***.***:

**Reply by guggero** [reply]
This looks good. Just make sure when copy/pasting there is no line break in the command (all in one single line). Put it into a text editor first, check that everything is in a single line, then copy it into the terminal.

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

Thank you!

I will try that and let you know how it goes.

I appreciate you taking the time out from your busy schedule to help us.

Orange Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



9 Jun 2023, 18:36 by ***@***.***:

**Comment by BonzoDogDooDah** [comment]
Hello Oliver,

I am sorry to bother you but when i followed your info and made this command all one line in a text editor and then used Lightning Shell in Umbrel what you see below is what happened after I ran that command.

Also I removed the :1 and added :0 and it still gave the same result?

Please let me know what I nee to do now. 

Is it possible that after using chantools and doing the doublespend with a new transaction we are using the wrong channel point of:

 23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6

Than you for any assistance you can offer us.

Warm regards,

Orange _Bonzo

***@***.***:~$ lncli abandonchannel--chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1--i_know_what_i_am_doing
No help topic for 'abandonchannel--chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1--i_know_what_i_am_doing'

--
 
 Sent with Tutanota, enjoy secure & ad-free emails. 



9 Jun 2023, 18:36 by ***@***.***:

**Reply by guggero** [reply]
You need a space between each part of the command. Just make sure it doesn't insert a newline. So copy this: `lncli abandonchannel --chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1 --i_know_what_i_am_doing`

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

Sorry for the bother yet again but this is what is is saying now.

--
***@***.***:~$ lncli abandonchannel --chan_point=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6:1 --i_know_what_i_am_doing
Incorrect Usage: flag provided but not defined: -chan_point

NAME:
   lncli abandonchannel - Abandons an existing channel.

USAGE:
   lncli abandonchannel [command options] funding_txid [output_index]

CATEGORY:
   Channels

DESCRIPTION:
   
  Removes all channel state from the database except for a close
  summary. This method can be used to get rid of permanently unusable
  channels due to bugs fixed in newer versions of lnd.

  Only available when lnd is built in debug mode. The flag
  --i_know_what_im_doing can be set to override the debug/dev mode
  requirement.

  To view which funding_txids/output_indexes can be used for this command,
  see the channel_point values within the listchannels command output.
  The format for a channel_point is 'funding_txid:output_index'.

OPTIONS:
   --funding_txid value      the txid of the channel's funding transaction
   --output_index value      the output index for the funding output of the funding transaction (default: 0)
   --i_know_what_i_am_doing  override the requirement for lnd needing to be in dev/debug mode to use this command; when setting this the user attests that they know the danger of using this command on channels and that doing so can lead to loss of funds if the channel funding TX ever confirms (or was confirmed)
   
[lncli] flag provided but not defined: -chan_point

--

Thank you!

Orange_Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



23 Jun 2023, 17:45 by ***@***.***:

**Reply by guggero** [reply]
Ah, then you are on an old version. Try `lncli abandonchannel 23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6 1 --i_know_what_i_am_doing`

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

I tried this and it may have worked but only using :0 instead of :1

Lightning Shell in Umbrel paused for a long time then said reconnecting to umbrel?

So it may have worked I am not sure?

Any ideas?

Again we really appreciate all of your help we could not have done it with out you.

Orange_Bonzo

--
lncli abandonchannel --funding_txid=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6 -- output_index=1 --i_know_what_i_am_doing

--​

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



23 Jun 2023, 18:33 by ***@***.***:

**Reply by guggero** [reply]
I'm not sure. Since I don't know what the channel point actually is, I can't tell you what value to use. Does the channel still show as pending in the Lightning app?

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

Thank you for the ongoing help, it is most appreciated.

--

lncli abandonchannel 
 --funding_txid=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6 
 -- output_index=1 --i_know_what_i_am_doing

All in one line with spaces and added a :0 instead of :1 and it seemed to work it just said reconnecting then the node got stuck for ages then I restarted umbrel and it all came back up online now with all channnels green and active.

Just that pending channel still saying ending?

Do we just wait a few hours or more for the command to finalize its job?

What is the usual way this command works? as the channtools command you gave to us a short while ago did work and almost straight away with the funds / sats being returned to the wallet very quickly?

Thank you!

Orange_Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



23 Jun 2023, 19:16 by ***@***.***:

**Comment by BonzoDogDooDah** [comment]
Hi again,

Sorry Oliver I checked the channel point and it is the same in RTL and LNDG and Thunderhub so that was correct so not sure if I need to run that abandonchannel command again?

That command did seem to knock out the system and needed to restart Umbrel so that was a relief.

All online and active just still with that nicehash channel channel point still pending?
Thank you for all of you hard work on our behalf.

Orange_Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



23 Jun 2023, 19:16 by ***@***.***:

**Reply by guggero** [reply]
Yes, then try the command again. And before trying out random permutations, report back what the error is if it doesn't work.

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

Ok, thank you kindly.

When you can could you please let me know what these actions mean?
So if the command below gave no errors after running it except for a bit of a long delay before stating that lightning shell is going to reconnect and then the node kind of froze and lnd would not reload getting stuck half way and would not come back up without us then doing a reboot mean anything to you?
--
lncli abandonchannel --funding_txid=23ef6674afb665350ca80eeb77af61e5cc414538297e08f07a7c251ebbcc18d6 --output_index=0 --i_know_what_i_am_doing
--
I assume no errors meant the abandonchannel command was doing something, just not sure if then lnd went buggy and just crashed / failed?

Thank you again for your help.

Orange_Bonzo


-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



23 Jun 2023, 19:41 by ***@***.***:

**Reply by guggero** [reply]
The command won't give you a response, other than just `{}` on the command line. The node should however not shut down, that seems to be a different problem. Check the log files (might need to consult the Umbrel guides to figure out how to do that).

**Comment by BonzoDogDooDah** [comment]
Hi Oliver,

Wow, I guess you do not need as much sleep as most of us do?

I will try that command again soon.
Thank you for your continued support, we appreciate all of your help.

Orange_Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



23 Jun 2023, 23:04 by ***@***.***:
